<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qsldb</class>
 <widget class="QMainWindow" name="qsldb">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QSL Datenbank</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>0</y>
      <width>53</width>
      <height>49</height>
     </rect>
    </property>
    <property name="text">
     <string>QSL</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonExit">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>424</y>
      <width>433</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Ende</string>
    </property>
   </widget>
   <widget class="QTreeWidget" name="cardList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>64</y>
      <width>271</width>
      <height>345</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>246</green>
          <blue>221</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>246</green>
          <blue>221</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>244</red>
          <green>244</green>
          <blue>244</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="rootIsDecorated">
     <bool>false</bool>
    </property>
    <column>
     <property name="text">
      <string> Call</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string> Datum</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Band</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Mode</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Qsl</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string> qsoId</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="cardLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>72</y>
      <width>505</width>
      <height>325</height>
     </rect>
    </property>
    <property name="text">
     <string>QslCard</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labeldbInfo">
    <property name="geometry">
     <rect>
      <x>316</x>
      <y>28</y>
      <width>231</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>428</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string> Call :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>420</y>
      <width>131</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labeldbSize">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>28</y>
      <width>241</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>28</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>64</red>
          <green>52</green>
          <blue>133</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>64</red>
          <green>52</green>
          <blue>133</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>146</red>
          <green>145</green>
          <blue>144</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>eQsl Karten Ordner</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
